### DISCUSSION TITLE
Using bzlmod for Android & iOS cross compile.

### DISCUSSION THREAD

**Discussion by widiba03304** [bazelbuild/bazel#22074] [Q&A]
Hey there, I'm trying to use bzlmod to cross-compile for native Android and iOS platforms. However, I'm having trouble finding any tutorials or documentation that match my specific needs. Can you guys assist me with this?

Regarding Android, it seems that rules_android and rules_android_ndk are available in the Bazel central library, but their releases are outdated or contain bugs. Do you happen to have any working examples for this scenario?

**Comment by meteorcloudy** [comment]
Did https://github.com/bazelbuild/rules_android?tab=readme-ov-file#getting-started work?

I believe rules_android is still actively work in progress, you probably need to use a newer commit via `git_override`. /cc @ted-xie
